gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oauth2/service/response/jwt/JWTAuthorizationResponse.java

```diff
@@ -159,4 +160,11 @@ public String buildRedirectUri() throws URISyntaxException {
 
         return uriBuilder.buildString();
     }
+
+    @Override
+    public MultiMap params() {
+        MultiMap result = MultiMap.caseInsensitiveMultiMap();
+        result.add(io.gravitee.am.common.oidc.Parameters.RESPONSE, token);
```

Comment:
Suggestion: 
there can be used a static import io.gravitee.am.common.oidc.Parameters.RESPONSE